frontend/src/static/css/panels.scss

```diff
@@ -90,6 +90,16 @@
 .toolbar {
   cursor: pointer;
   float: right;
+
+  &--multiline {
```

Comment:
Just a question, why we need to move the CSS here?

Reply:
This `toolbar--multiline` CSS is used by both zoom and authorship page. Moved here to avoid duplication of code in separate places.

Reply:
hmm, why is it named multiline?

Reply:
This allows the links on the toolbars to appear on multiple lines instead of one line. Example: 
![image](https://user-images.githubusercontent.com/5328196/81406335-f2d82080-916b-11ea-8cd9-4dd13a1393ce.png)